pkg/shell/plot.js

```diff
@@ -352,7 +353,13 @@ shell.plot = function plot(element, x_range_seconds, x_stop_seconds) {
                                             limit: (x_offset > 0) ? (x_range / interval) : undefined,
                                             host: desc.host
                                           });
-        $(archive_channel).on("message", handle_message);
+        $(archive_channel).on("message", function(data) {
+            if (first) {
+                first = false;
+                $("#dashboard-toolbar").show();
```

Comment:
This couples the plot gadget to the dashboard, which is obviously non-optimal. We could pass the selector in as an option.

(The need for this might change with #1767, which provides a in-browser 'archive' of the received real-time samples.)